### ISSUE TITLE
Error using Timer in TypeScript

### ISSUE THREAD

**Issue by RobinDijkstra** [volkov97/react-compound-timer#4]
Hi,
I'm working on a React TypeScript project. I've added your package (version 1.0.12) like this:

`npm i react-compound-timer --save`

I've added an import to my App.tsx:

`import Timer from 'react-compound-timer';`

And for testing purposes, I've added the simplest implementation of the Timer component:

```
<Timer>
   <Timer.Seconds />
</Timer>
```

I'm getting the following error:

```
JSX element type 'Timer' is not a constructor function for JSX elements.
  Type 'Timer' is missing the following properties from type 'ElementClass': context, setState, forceUpdate, props, and 2 more.ts(2605)
(alias) class Timer
import Timer
```

Am I missing something? Has this something to do with recent Typescript support?

**Comment by volkov97** [comment]
Hi, can you tell me if it is working correctly in 1.0.11 ?

**Comment by volkov97** [comment]
I created a repo for example.

This commit illustrates how to add my library!

https://github.com/volkov97/react-compound-timer-example/commit/0a47e3739f34cba2be5821f6015624d10f448628

**Comment by RobinDijkstra** [comment]
1.0.11 didn't work.
The steps in your example were exactly the same I did. To be sure, I've created a new, empty project and added the library. Same problem.
Then I noticed differences between your example code and mine and I wondered what was different in creating them. I've created mine with

`create-react-app my-pwa-app --scripts-version=react-scripts-ts`

and after a some searching I discovered that this way is deprecated (https://github.com/wmonk/create-react-app-typescript) because create-react-app now supports typescript natively. I missed that. So I created a new project with

`create-react-app my-pwa-app --typescript`

and added your library. Everything is working fine now. Thank you for your example code and keep up the good work!
